debugging-w-chatgpt/debugging-w-chatgpt.md

```diff
@@ -325,27 +325,26 @@ Once we’ve found and fixed the bug, we need to test our code to make sure the
 
 As stated before, we can’t wholeheartedly trust everything ChatGPT says. Evaluating a code response from ChatGPT should look pretty similar to evaluating any other information created by a generative AI - we need to look for verifiable pieces of information in the response and use other trustworthy sources to confirm if those pieces of information are true. Check out the lesson “Prompting Basics & Using ChatGPT - Verifying Information from ChatGPT” if you’d like to refresh yourself on strategies for verifying information.
 
-Let’s say that a student, Althea, is working on [the Adagrams project](https://github.com/AdaGold/adagrams-py) and is debugging an `IndexError` that sometimes appears in their `draw_letters` function:
+Let’s say that a student, Althea, is working on a game with a function that draws 7 random domino tiles from a pool. She is debugging an `IndexError` that sometimes appears in their `draw_tiles` function:
```

Comment:
```suggestion
Let’s say that a student, Althea, is working on a game with a function that draws 7 random domino tiles from a pool. They are debugging an `IndexError` that sometimes appears in their `draw_tiles` function:
```

Not super important to the content, but through this lesson I used exclusively they/them pronouns for the example student Althea.

Reply:
I missed this! I'll get it updated!